### DISCUSSION TITLE
178

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#144] [General]
# 潮流周刊第178期 - 蜗牛蜗牛

封面图拍摄来源于上周在路上拍摄到的一只蜗牛，刚好地面是一个蓝白色分割面，天气不热，蜗牛不急不慢的走着，挺有感觉。

https://weekly.tw93.fun/posts/178-%E8%9C%97%E7%89%9B%E8%9C%97%E7%89%9B

**Comment by umuzhi** [comment]
up主，你好呀，很喜欢你的作品和文章。另外，如果博主有时间和有兴趣的时候能不能吧潮流周刊也支持下rss订阅，谢谢，个人想法哈哈

**Reply by tw93** [reply]
其实是支持的 在首页右上角 你去看看  https://weekly.tw93.fun/rss.xml

**Reply by umuzhi** [reply]
thank you body

**Comment by hualet** [comment]
好奇博主是怎么能每周发现这么多有意思的产品的？

**Reply by tw93** [reply]
每个周末有一个固定下午或者晚上学习时间

**Comment by huyixi** [comment]
这期的封面太漂亮了
